ert_gui/tools/ide/ide_tool.py

```diff
@@ -7,18 +9,19 @@
 
 
 class IdeTool(Tool):
-    def __init__(self, path, help_tool):
+    def __init__(self, config_file, help_tool):
         super(IdeTool, self).__init__("Configure", "tools/ide", resourceIcon("ide/widgets"))
 
         self.ide_window = None
-        self.path = path
+        self.config_file = config_file
+        self.path = basename(config_file)
         self.help_tool = help_tool
 
     def trigger(self):
         if self.ide_window is None:
-            self.ide_window = ref(IdeWindow(self.path, self.parent(), self.help_tool))
+            self.ide_window = ref(IdeWindow(self.config_file, self.path, self.parent(), self.help_tool))
```

Comment:
Sorry for being picky - but i see now the path is not used in this file either... could you just pass on the config-file and make ide-window get the path where it is actually used?